src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,57 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String OPERATION_BUY = "buy";
+    private static final String OPERATION_SUPPLY = "supply";
 
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static String createReport(String[] dataArray) {
+        int amountBuy = 0;
+        int amountSupply = 0;
+        for (int i = 0; i <= dataArray.length - 2; i += 2) {
```

Comment:
calculations are not the part of creating report. create separate method or calculate what you need in `getStatistic` method